docs/graphql/core/scheduled-triggers/create-one-off-scheduled-event.rst

```diff
@@ -131,3 +131,11 @@ Once you've created your scheduled trigger, you can see ``Pending events``, ``Pr
 .. thumbnail:: /img/graphql/core/event-triggers/pending-one-off.png
    :alt: Schedule and logs for scheduled events
    :width: 1200px
+
+
+One minute precision
```

Comment:
Not sure if we need a separate section for this. Maybe we can make it a subsection in the "Schedule & logs" section, since it's actually about the schedule?

Reply:
Great